R/convert_ov_wenr.R

```diff
@@ -0,0 +1,37 @@
+#' Convert possible B_OV_WENR values to standardised values
+#' 
+#' This function converts numeric values for OV_WENR to values used by other OBIC functions if numeric values are entered.
+#' 
+#' @param B_OV_WENR () Data on soil compaction risk that may have to be converted to string
+#' 
+#' @import data.table
+#' 
+#' @export
+
+convert_WENR <- function(B_OV_WENR) {
```

Comment:
Maybe renaming function to convert B_OV_WENR is more clear. As WENR can be supplier of more variables